.github/workflows/go.yml

```diff
@@ -20,6 +20,10 @@ env:
   GOMODCACHE: /home/runner/go/mod
   GOPROXY: https://proxy.golang.org
 
+  # Temporary workaround for https://github.com/codecov/codecov-action/issues/837
+  CODECOV_TOKEN: 1e257034-a484-4a13-abdf-009e769b909b
```

Comment:
I'm sure that's not a mistake, but to be 110% sure I need a confirmation that the upload token can be shared :wink: